client/src/utils/api.js

```diff
@@ -410,3 +410,18 @@ export const editResourceCategories = (
       },
     );
 };
+
+export const reportTranslationError = (
+  id: string,
+  language: string,
+  type: string,
+): ApiResponse<void> => {
```

Comment:
Do you wanna make a flow type for the response and add it here instead of void?

Reply:
ye